server/src/main/java/com/hedera/block/server/persistence/storage/PersistenceStorageConfig.java

```diff
@@ -146,4 +169,20 @@ public enum StorageType {
          */
         NO_OP
     }
+
+    /**
+     * An enum that reflects the type of compression that is used to compress
+     * the blocks that are stored within the persistence storage.
+     */
+    public enum CompressionType {
+        /**
+         * This type of compression is used to compress the blocks using the
+         * `Zstandard` algorithm.
+         */
+        ZSTD,
+        /**
+         * This type means no compression will be done.
+         */
+        NONE
+    }
```

Comment:
To go along with the previous suggestion:
```suggestion
    public enum CompressionType {
        /**
         * This type of compression is used to compress the blocks using the
         * `Zstandard` algorithm.
         */
        ZSTD(0, 20),

        /**
         * This type means no compression will be done.
         */
        NONE(Integer.MIN_VALUE, Integer.MAX_VALUE);

        private final int min;
        private final int max;

        private CompressionType(final int minValue, final int maxValue) {
            min = minValue;
            max = maxValue;
        }

        public void verifyCompressionLevel(final int levelToCheck) {
            Preconditions.requireInRange(levelToCheck, min, max);
        }
    }
```

Reply:
✅ Done.